projects/vdk-plugins/vdk-control-api-auth/setup.py

```diff
@@ -25,14 +25,9 @@
     "authentication.",
     long_description=pathlib.Path("README.md").read_text(),
     long_description_content_type="text/markdown",
-    install_requires=[],
+    install_requires=["requests"],
```

Comment:
why not reusing vdk-core's configuration mechanism?

Reply:
To not rely on vdk-core and its dependencies. I am hoping for the library to be as light as possible, as we will use it for the airflow provider.

Reply:
the airflow provider is not vdk-core-dependant?

Reply:
Not currently, https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-plugins/airflow-provider-vdk/setup.py#L28
Not sure if there are plans to have vdk-core as a dependency.